app/prisma/schema.prisma

```diff
@@ -864,3 +865,18 @@ model CachedProcessedEntry {
     @@index([incomingInputHash, nodeId])
     @@index([projectId])
 }
+
+model ExternalModel {
+    id String @id @default(uuid()) @db.Uuid
+
+    name String
+    displayName String
+    endpoint  String
+    apiKey  String
```

Comment:
For things like calculating the price of evals, won’t we need to know price per input and output token?

Reply:
Yes, I don't think we can't get the price from the endpoint/model name. We could ask the client to add this information as an option, but we cannot keep it up to date. 
The current logic does not show the price for external models added by a user.
Do you think we should add an option to a user to enter the price per token?

Reply:
I know that some other companies ask the user to input token prices, and it seems to work well for them.

Reply:
Sure, I will add it then! It might be useful in certain scenarios.